extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -160,6 +160,12 @@ export class SafeframeHostApi {
     /** @private {?({width, height}|../../../src/layout-rect.LayoutRectDef)} */
     this.creativeSize_ = creativeSize;
 
+    /** @private {?({width, height}|../../../src/layout-rect.LayoutRectDef)} */
+    this.initialCreativeSize_ = {
+      height: creativeSize['height'],
```

Comment:
Why access via quotes?  Can this just be this.initialCreativeSize_ = Object.assign({}, this.creativeSize);

Also note that creative size can be null here... is that expected?